mkosi.images/build/mkosi.conf.d/centos-fedora/mkosi.conf

```diff
@@ -20,3 +20,4 @@ Packages=
         libasan
         libubsan
         rpm-build
+        which
```

Comment:
I think it is better to add a short comment why needed.

Reply:
Ouch, little bit late...

Reply:
I think the git blame is sufficient documentation in this case.